src/containers/AlbumsView.jsx

```diff
@@ -32,17 +32,18 @@ export class AlbumsView extends Component {
       return <ErrorComponent errorType='albums' />
     }
     return (
-      <div>
-        <Topbar viewName='albums' />
-        <AlbumsList {...this.props} {...this.state} />
-        { this.props.children }
-      </div>
+      !this.props.children
```

Comment:
Yep, if there is an albumId AND a photoId, the viewer will be rendered as a child as AlbumPhotos